src/css/application.css

```diff
@@ -341,7 +342,7 @@ body {
 }
 
 .editors__label {
-  background-color: var(--color-low-contrast-gray);
+  background-color: var(--color-medium-contrast-gray);
```

Comment:
@iamnickvolpe sorry to go back and forth so much on a two-line pull request! But I don’t think “medium contrast gray” is doing the descriptive work we need here. The idea behind “low contrast gray” is specifically to call out “you shouldn’t use this in a situation where it _needs_ to contrast with white for all users”. Otherwise I would propose putting a new gray color on the general spectrum of “light gray”, “gray”, “dark gray”. Where does this new gray fall on that spectrum? Can we give any hints about when it’s appropriate to use this gray instead of the four other ones available?